### ISSUE TITLE
Router v2 in Bypass mode, are -d stats possible?

### ISSUE THREAD

**Issue by InertiaImpact** [Tysonpower/starlinkstatus#27]
I have the Square Dishy with the V2 router (with ethernet adapter) providing my existing network with its WAN connection. 

I have the Starlink router in bypass mode, is it possible to still get -d stats with this or not because the router is truely "disabled"? (Even though I can view stats "remotely" through the app still)

Thanks!

**Comment by Tysonpower** [comment]
Yes this should be possible, when the stats are working over the App they should work over the script as well.

regards, Manuel

**Comment by InertiaImpact** [comment]
I have things setup on a Pi but this is what I'm getting when it runs with Dishy stats option. 

The router is in bypass mode and I am not able to reach 198.162.100.1 or dishy.starlink.com locally. (was prior, bypass stopped that)

`getting Dishy Data...
Failed to dial target host "192.168.100.1:9200": context deadline exceeded`

Any suggestions? Thanks

**Comment by Tysonpower** [comment]
okay then it seems that bypass mode kills that somehow...but the app should use a similar endpoint, so maybe the dish got a new Ip? But it is very hard for me to find out, if you get it working let me know tho :)

**Comment by InertiaImpact** [comment]
I just wanted to update this note that it is possible to get the Dishy Data connection working when in Bypass mode.

On the 3rd party router side of things you just need to set up a static route for it. I am not entirely sure why this works the way it does as Netgear has certain limitations on what your destination can be set to but currently I have it set to my WAN ip address. Now, it has not changed in a long while, I suspect when it changes next I will need to update it to keep the route working but since I cannot put 0.0.0.0 as other guides have suggested, the public IP works.

[Here is one of the writeups ](https://community.netgear.com/t5/Orbi/RBR50-with-StarLink/td-p/2190141)where I got most of the info I needed to piece together how to set a static route, different router though. Both the ip and dishy.starlink.com are accessible to me now as well as from the App when on the local network.
